### PR TITLE
emoji-datasource 5.0.1, remove emojione&messenger support, new categories

### LINKED ISSUE #269: Emoji version 11+
Are there any plans for adding newer Emojis?
Where are they pulled in from?

### PR DESCRIPTION
**Pull request by tompiard**
I've made a fork with new datasource version, not really sure what was your plan concerning this but it's working fine (except a few details) on this PR.
It removes support for emojione set.

- [x] Bump **emoji-datasource** to version 5.0.1
- [x] Remove sets : **emojione** (discontinued by datasource) and **messenger** (merged with facebook)
- [ ] Support new categories _Smileys & Emotion_ + _People & Body_ (missing icon)
- [ ] Support new emojis with dual skin color
- [ ] Update doc (need a recheck)
- [x] Tests

Closes #269

### CODE REVIEW

File: src/svgs/index.js [hunk 1]
```diff
@@ -101,6 +101,18 @@ const categories = {
     </svg>
   ),
 
+  smileys: () => (
```
Comment (tompiard) [inline src/svgs/index.js]: Need new icon here.

File: src/utils/index.js [hunk 2]
```diff
@@ -111,24 +111,26 @@ function getData(emoji, skin, set, data) {
   if (emojiData.skin_variations && skin > 1) {
     emojiData = _JSON.parse(_JSON.stringify(emojiData))
 
-    var skinKey = SKINS[skin - 1],
-      variationData = emojiData.skin_variations[skinKey]
+    var skinKey = SKINS[skin - 1]
+    if (emojiData.skin_variations[skinKey] !== undefined) {
+      var variationData = emojiData.skin_variations[skinKey]
 
-    if (!variationData.variations && emojiData.variations) {
```
Comment (tompiard) [inline src/utils/index.js]: Temporary fix for dual skin color emojis

File: stories/index.js [hunk 3]
```diff
@@ -93,6 +93,17 @@ storiesOf('Picker', module)
...
             </svg>
           ),
+          people: () => (
```
Comment (tompiard) [inline stories/index.js]: Change icon here

**Comment by EtienneLem** [comment]
Thanks a lot for this @tompiard, this has been merged in 5c083a8

**Pull request closed without merging**